### PR TITLE
Added test for area of code

### PR DESCRIPTION
**Pull request by kshitij3199**
Hi @valeriocos ,
Please have a look at the new test (test_enrich_areas_of_code) and let me know if something more has to be done.
When I run it in my laptop the convergence of git.py file increases from 44% to 49% :smile:

### CODE REVIEW

File: tests/test_git.py [hunk 1]
```diff
@@ -283,6 +283,7 @@ def test_enrich_forecast_activity(self):
         delete_survival = self.es_con + "/git_study_forecast_activity"
         requests.delete(delete_survival, verify=False)
 
+
```
Comment (valeriocos) [inline tests/test_git.py]: Please remove this empty line, thanks!

File: tests/test_git.py [hunk 2]
```diff
@@ -303,6 +304,82 @@ def test_onion_study(self):
...
+        self.assertIn('origin', source)
+        self.assertIn('owner', source)
+        self.assertIn('perceval_uuid', source)
+        self.assertIn('project', source)
+        self.assertIn('project_1', source)
+        self.assertIn('removedlines', source)
+        self.assertIn('repository', source)
+        self.assertIn('uuid', source)
+
+        source = response['git_aoc-enriched']['settings']['index']
```
Comment (valeriocos) [inline tests/test_git.py]: Lines 374 - 379 aren't needed, can you remove them?

**Review by valeriocos — COMMENTED** [review]
Great @kshitij3199 ! Please squash the commits in one, and check the minor comments when you have time

Travis is under maintenaince, I would wait until is back to merge the PR.

Thank you for your work!

**Comment by kshitij3199** [comment]
Thank you @valeriocos,
I will update test_git.py file as per your instructions. And will send a PR when Travis is working.

**Comment by valeriocos** [comment]
Travis is now working @kshitij3199 , can you check the errors? thanks

**Comment by kshitij3199** [comment]
Hi @valeriocos , 
I have send a new PR #816. Please have a look.
Travis check is not happening for that PR.

**Comment by valeriocos** [comment]
Hi @kshitij3199, which PR should I check? This one, #816 or #817. Please close the ones that you want to discard, thanks!

**Comment by kshitij3199** [comment]
Hi @valeriocos ,
Sorry for confusion. Please check PR #817

**Comment by valeriocos** [comment]
No worries, thanks!

**Pull request closed without merging**